rust/sdk-examples/src/db/common/models/backfill_processor_status.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::extra_unused_lifetimes)]
+
+use crate::{schema::backfill_processor_status, utils::database::DbPoolConnection};
+use diesel::{AsChangeset, ExpressionMethods, Insertable, OptionalExtension, QueryDsl, Queryable};
+use diesel_async::RunQueryDsl;
+
+#[derive(AsChangeset, Debug, Insertable)]
+#[diesel(table_name = backfill_processor_status)]
+/// Only tracking the latest version successfully processed
+pub struct BackfillProcessorStatus {
+    pub processor_name: String,
+    pub last_success_version: i64,
+    pub last_transaction_timestamp: Option<chrono::NaiveDateTime>,
+    pub backfill_start_version: i64,
+    pub backfill_end_version: i64,
```

Comment:
seems like this isn't explicit being used, is this to track when to stop?

Reply:
These new `backfill_X_version` fields are only to be used to determine backfill progress. They're a convenience to be seen in the postgres table rather than having to be checked in the config every time!